docusaurus/docs-glossary/glossary.md

```diff
@@ -4,12 +4,19 @@ id: glossary
 # Glossary
 Here you will find a list of terms you may come across as you are using Ziti.
 
+## Edge Router Policy
+An Edge Router Policy is a policy/configuration that describes which Identities can connect to which Edge Routers
+
 ## Network Overlay, Overlay
 A Ziti network is implemented as an "overlay". A network overlay abstracts away the layers beneath it, providing a new set of abstractions for designing and implementing software and systems. Good programming abstractions allow developers to focus on the rules implemented by those abstractions without being concerned with the layers below the abstraction. Ziti's overlay allows developers to focus on connectivity between components without having to be concerned with low-level details of how that connectivity is managed.
 
 ## Service Definition
 A service definition is used to "bind" a service to a specific underlay network expression, through one or more nodes on a Ziti overlay network. A service definition usually includes a terminating router (or routers) and one or more SDK or underlay network endpoints where the service can be reached.
 
+## Service Edge Router Policy
+A Service Edge Router Policy is a policy/configuration that describes which Edge Routers are able to be used for inbound
+or outbound traffic for a particular Service

```

Comment:
"inbound and outbound" is a little more clear because "or" implies that the SERP only applies to one or the other for a particular service.